layers.go

```diff
@@ -112,33 +112,33 @@ type Layer struct {
 	Created time.Time `json:"created,omitempty"`
 
 	// CompressedDigest is the digest of the blob that was last passed to
-	// ApplyDiff() or Put(), as it was presented to us.
+	// ApplyDiff() or put(), as it was presented to us.
```

Comment:
`Put()` is no more (a great cleanup), so these references should probably point to `create()`, throughout.

Reply:
Oof, of course you're right.  Fixing.